autogpt_platform/backend/backend/notifications/conftest.py

```
@@ -0,0 +1,79 @@
+from typing import Iterator
+from unittest.mock import AsyncMock
+
+import pytest
+
+NOW = datetime(2026, 8, 3, 7, 30, tzinfo=timezone.utc)
+
+
+@pytest.fixture(scope="session")
+def server() -> None:
+    """The notification suite is pure logic; it needs no live stack."""
+    return None
```

Ruff (0.16.2)

[warning] Do not explicitly `return None` in function if it is the only possible return value

Remove explicit `return None`

(RET501)